src/index.ts

```diff
@@ -12,21 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { PolicyPack } from "@pulumi/policy";
+import { AwsGuard, AwsGuardArgs } from "./awsGuard";
 
-import { compute } from "./compute";
-import { database } from "./database";
-import { elasticsearch } from "./elasticsearch";
-import { security } from "./security";
-import { storage } from "./storage";
+// Import each area to add AwsGuardArgs mixins and register policies.
+import "./compute";
+import "./database";
+import "./elasticsearch";
+import "./security";
+import "./storage";
 
-// Create a new Policy Pack.
-export const policyPack = new PolicyPack("pulumi-awsguard", {
-    policies: [
-        ...compute,
-        ...database,
-        ...elasticsearch,
-        ...security,
-        ...storage,
-    ],
-});
+export { AwsGuard, AwsGuardArgs };
+
+// If we're running our integration tests, create a new instance of AwsGuard.
+if (process.env["PULUMI_AWSGUARD_TESTING"] === "true") {
+    const awsGuard = new AwsGuard();
+}
```

Comment:
I've been thinking about whether we want to provide a built-in way that news-up the policy pack from within this library (and I saw what Chris did [here](https://github.com/pulumi/pulumi-awsguard/pull/31/files#diff-f41e9d04a45c83f3b6f6e630f10117fe)), and the more I think about it, the more I think we shouldn't do it. Doing it would be the equivalent of having a resource library like `@pulumi/eks` that had some way of newing up an `eks.Cluster` within the library itself, rather than requiring you to create your own Pulumi program that does it.

So my feeling is, we should just encourage the creation of your own Policy Pack (it'll be really easy to do because I'll be adding a template for `pulumi policy new`).

However, given the way we've currently created our integration tests, we just pass the src directory to `--policy-pack` when running the preview, so to keep the integration tests working in the simplest way, for now, I've used this temporary approach. But I'd like to follow-up and remove this from the library and fix-up the integration tests to not do it this way. I can open an issue and add a TODO if you agree.

Reply:
I personally want to disagree. Because needing users to create their own, new, Node module that does nothing but instantiate a `new AwsGuard()` is kinda lame. (Since I need to get the files on disk, run `npm install`, etc.)

But even in a world where you can use a "built-in" or "default" version of the AWS Guard policy pack, you still need to have the source code on disk anyways. So even then we aren't really saving you any trouble.

So I'm inclined to agree with you. And that we should encourage the creation of custom Policy Packs, and just make that easy via `pulumi policy new`.